packages/paste-website/src/pages/introduction/contributing/components/index.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Contributing a component
+description: How to contribute a component to Paste.
+slug: /introduction/contributing/components/
+---
+
+import {graphql} from 'gatsby';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/introduction/contributing/components/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+How to contribute a component to Paste.
+
+---
+
+Thanks for your interest in contributing a component! There are many ways to contribute and it’s up to you what scope and type of contribution will have the most impact.
+
+The following guidelines can flex to your capacity, so feel free to adapt as you see fit.
+
+## Ways to contribute
+
+- Contributing the code for a new component
+- Creating a variation of a component
+- Adding design assets of a component to Figma
+- Documenting one part of a component
+- Sharing research related to a component
+- Refining an existing component based on new information
+
+## Roles
+
+### Design System’s role
+
+We’re here to partner with you throughout the component process to ensure you’re building something that is systemized and accessible. You’ll be paired with a designer and engineer from our team to facilitate the process.
+
+We’ll also help you refine design assets for our Figma libraries, build the component in code, and add the spec content to the docs site.
+
+### Your role (anyone at Twilio!)
+
+You’ll create and research the component and socialize it within your crit pods, committees, and peers.
+
+It’s helpful for designers and engineers to pair together on this work to understand the product-specific customer problems, and implementation on both sides.
+
+## Process
+
+### General process
+
+This is the process we follow when creating a net-new component for Paste. This process is fluid and adaptable to the scope and type of your contribution.
+
+1. Start by filling out the kickoff section of a [component spec](https://docs.google.com/document/d/1e1m4amg1Khbxo-limb-PPDSJvNeDF2YBFCWrqeS5MPc/edit). Here are some [examples of other specs](https://drive.google.com/drive/folders/1ehHLxeX3OBpgW5TkGc0KkmAAFEnGj4L3) people have written.
```

Comment:
Is this page public for all Paste customers? If yes, are these documents Twilio employee only access?

Reply:
Yep it's public and assuming the docs are Twilio protected. I'm guessing that's fine because higher up in the page we say "anyone at Twilio" can contribute. @loreina what do you think?

Reply:
For this part, I was following how we had it in the Contributing a pattern page. It should be _fine_ that they're protected for Twilio internal use and maybe we can look into modifying these contributing pages for external customers in the future. Could even have 2 paths for contributing depending on if you're internal or external.